sonar-plugin-api/src/main/java/org/sonar/api/test/MutableTestCase.java

```diff
@@ -23,6 +23,10 @@
 import javax.annotation.Nullable;
 import org.sonar.api.batch.fs.InputFile;
 
+/**
+ * @deprecated since 5.6 feature will be removed soon
+ */
+@Deprecated
 public interface MutableTestCase extends TestCase {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1133)